src/udiskslinuxfilesystem.c

```diff
@@ -183,6 +184,15 @@ udisks_linux_filesystem_update (UDisksLinuxFilesystem  *filesystem,
   dev = udisks_linux_block_object_get_device_file (object);
   type = g_udev_device_get_property (device->udev_device, "ID_FS_TYPE");
   size = 0;
+
+  /* skip udisks_linux_filesystem_update when ata drive is inactive */
+  if (device != NULL)
+    {
+      if (g_udev_device_get_property_as_boolean (device->udev_device, "ID_ATA"))
+             if (!udisks_linux_drive_ata_is_active (device, &error))
```

Comment:
In this case `error` should be ignored, you don't want to fail the whole function (it's not tested anywhere anyway) and it's not necessary to have this set. Please pass `NULL` instead.